vm/debugging.cpp

```diff
@@ -0,0 +1,66 @@
+// vim: set sts=2 ts=8 sw=2 tw=99 et:
+// 
+// Copyright (C) 2016-2018 AlliedModders LLC
+// 
+// This file is part of SourcePawn. SourcePawn is free software: you can
+// redistribute it and/or modify it under the terms of the GNU General Public
+// License as published by the Free Software Foundation, either version 3 of
+// the License, or (at your option) any later version.
+//
+// You should have received a copy of the GNU General Public License along with
+// SourcePawn. If not, see http://www.gnu.org/licenses/.
+//
+#include "debugging.h"
+#include "stack-frames.h"
+#include "environment.h"
+#include "watchdog_timer.h"
+#include <amtl/am-raii.h>
+
+namespace sp {
+
+void InvokeDebugger(PluginContext* ctx, const IErrorReport* report)
+{
+  // Ignore any calls if this isn't enabled.
+  if (!Environment::get()->IsDebugBreakEnabled())
```

Comment:
Is it possible to get here if the debug break is not enabled?

Reply:
Yes, through the interpreter.